packages/devextreme/js/ui/switch.js

```diff
@@ -316,17 +316,17 @@ const Switch = Editor.inherit({
         });
     },
 
-    _renderValue: function() {
+    _renderValue() {
         this._validateValue();
 
-        const val = this.option('value');
-        this._renderPosition(val, 0);
+        const { value } = this.option();
```

Comment:
I think this approach can affect on Knockout integration. Perhaps, you will remember: [link](https://teams.microsoft.com/l/message/19:d52fea9c6ac94ea1abc470486f4a963a@thread.skype/1686051486996?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=8c31b3e5-5870-46b1-92a7-180d13b282ed&parentMessageId=1686051486996&teamName=DevExtreme&channelName=Components%20Squad%20(English)&createdTime=1686051486996)

Reply:
[389c8403d64d10319b382630250b40d2cf9e5d06]